Python pokemon/generate_team.py

```diff
@@ -0,0 +1,46 @@
+import requests
+import random
+from pydantic import BaseModel
+import json
+
+class Pokemon(BaseModel):
+    name: str
+    types: list[str]
+    hp_stat: int
+    ability_names: list[str]
+
+team = []
+
+for _ in range(6):
+    response = requests.get(f"https://pokeapi.co/api/v2/pokemon/{random.randrange(1, 150)}/")
+    data = response.json()
+
+    # Extract values
+    name = data["name"]
+    hp_stat = next(stat["base_stat"] for stat in data["stats"] if stat["stat"]["name"] == "hp")
+    types = [type_entry["type"]["name"] for type_entry in data["types"]]
+    ability_names = [ability["ability"]["name"] for ability in data["abilities"]]
+
+    # Initialize with all required fields
+    temp_poke = Pokemon(
+        name=name,
+        types=types,
+        hp_stat=hp_stat,
+        ability_names=ability_names
+    )
+
+    team.append(temp_poke)
+
+team_data = [poke.model_dump() for poke in team]
+
+#Sort pokemons based on hp_stat
+def get_pokemon(element):
+    return element['hp_stat']
+
+team_data.sort(key=get_pokemon)
+
+# Save to a file
+with open("Python pokemon/team.json", "w") as file:
+    json.dump(team_data, file, indent=2)
```

Comment:
Helt fint, eneste jeg kan sige er at du kunne tilføje en try/except til at håndtere fejl, da netop sådan noget som fil håndtering er, IO operationer, kan lettere skabe problemer og blive påvirket end meget andet

```suggestion
try:
    with open("Python pokemon/team.json", "w") as file:
        json.dump(team_data, file, indent=2)
except (OSError, IOError) as e:
    print(f"An error occurred while saving the file: {e}")
except Exception as e:
    // Tilføjet generisk exception, da det altid er godt og prøve at håndtere selv de fejl man ikke lige har tænkt på. Den her fanger alle andre exceptions end OSError og IOError foroven
    print(e)
```